crates/sui-cost/tests/calibration.rs

```diff
@@ -35,6 +36,7 @@ async fn test_bytecode_disassemble_snapshot() -> Result<(), anyhow::Error> {
 }
 
 fn disassemble_test_module(name: String) -> anyhow::Result<String> {
+    move_package::package_hooks::register_package_hooks(Box::new(SuiPackageHooks));
```

Comment:
This test is an outlier in that it uses the package hooks but also `BuildConfig` from `move_package` rather than `sui_move_build`.  Are there other discrepencies that can creep in due to that difference?